docs/Status_Reference.md

```diff
@@ -242,6 +242,8 @@ The following information is available in the `gcode_move` object
 The following information is available in the
 [hall_filament_width_sensor](Config_Reference.md#hall_filament_width_sensor)
 object:
+- all items from
```

Comment:
I see the docs don't reference other status objects like this anywhere else, so I'm happy to rework it if necessary.

I tried to write out all the items explicitly, but got a bit stuck with how to concisely explain the "embedded" filament switch sensor.